app/controllers/car_models_controller.rb

```diff
@@ -0,0 +1,30 @@
+class CarModelsController < ApplicationController
+
+  def new
+    @manufacturer = Manufacturer.find(params[:manufacturer_id])
+    @car_model = CarModel.new
+  end
+
+  def create
+    @manufacturer = Manufacturer.find(params[:manufacturer_id])
+
+    if @car_model = @manufacturer.car_models.create(car_model_params)
```

Comment:
@car_model = @manufacturer.car_models.create(car_model_params) will always return a CarModel object
if the creation doesn't work, the @car_model will be invalid